test/test_grammar.js

```diff
@@ -94,7 +94,22 @@ describe('grammar', () => {
         const [result] = parser.parse('1. e4 (1. d4 d5) e5 2. d4 exd4 *');
         result.moves.should.have.lengthOf(4);
     });
+
+    it('should allow move after RAV without whitespace', () => {
+        const [result] = parser.parse('1. e4 (1. d4 d5)e5 2. d4 exd4 *');
+        result.moves.should.have.lengthOf(4);
+    });
+
+    it('should allow RAV after RAV without whitespace', () => {
+        const [result] = parser.parse('1. e4 (1. d4 d5)(1. f4 e5)e5 2. d4 exd4 *');
+        result.moves.should.have.lengthOf(4);
+    });
     
+    it('should allow promotion without equal sign', () => {
```

Comment:
Not necessarily opposed to this change but I can't find it anywhere in the PGN spec.

Reply:
Indeed, I could not find them in the PGN spec neither. I've just seen that some tools (such like chessX) or webistes (chess.com it seems to me) generated PGN this way. I don't know why, but I've already encountered such format.